Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -71,6 +71,15 @@ The example below uses, a ***Pre Heading***, a ***Heading***, a ***Text*** and t
 
 The Gallery Widget allowes you to display multiple images in a gallery view, meaning that the images will be alligned in rows. 
 This also contians the feature to click on images to zoom in on the images(this feature can also be disabled).
+You will be able to change the set up of your gallery for example the size of the images and the background color of the whole container. 
+Below you can see how the gally would look with image width 20%, and a blue background color.
```

Comment:
When you want a linebreak, please make sure to add an extra break. In the case here, it will all be in the same section without showing any linebreaks.